frontend/src/store/modules/http/state.js

```diff
@@ -1,8 +1,7 @@
 export const TOKEN = 'TOKEN';
 
 export default {
-  // LEGACY SUPPORT
-  // After merge to new version remove localStorage
-  [TOKEN]: localStorage.getItem('jwt') || null,
+    // LEGACY SUPPORT
```

Comment:
that's just nitpicking, but in JavaScript world, indentation is usually multiplication of 2.

Reply:
Yeah, I know, I actually removed that code but then reverted it with wrong indentation, hence the change. Will fix it